app/src/main/java/com/example/drawingactivity/MainActivity.kt

```diff
@@ -1,20 +1,24 @@
 package com.example.drawingactivity
 
 import android.os.Bundle
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
 
+
 class MainActivity : AppCompatActivity() {
 
-    private val viewModel = CustomColorViewModel()
+    private val viewModel: CustomColorViewModel by viewModels()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val drawableArea = findViewById<DrawableAreaView>(R.id.drawableArea)
```

Comment:
Where is the lambda?@!?@!?

Reply:
lamb who?

in all seriousness I missed removing it when I realized that saving and loading the bitmap from the viewModel wasn't a requirement for this ticket. Will look forward into how to properly implement that in the future.